troposphere/static/js/components/common/InstanceHistorySection.react.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import Backbone from 'backbone';
+import stores from 'stores';
+import moment from 'moment';
+
+var InstanceHistorySection = React.createClass({
+    displayName: "InstanceHistorySection",
+
+    propTypes: {
+        instance: React.PropTypes.instanceOf(Backbone.Model).isRequired
+    },
+
+    getInitialState: function(){
+        return{
+            instanceHistory: stores.InstanceHistoryStore.fetchWhere({"instance": this.props.instance.id})
+        }
+    },
+
+    componentDidMount: function(){
+        stores.InstanceHistoryStore.addChangeListener(this.onNewData);
+    },
+
+    componentWillUnmount: function(){
+        stores.InstanceHistoryStore.removeChangeListener(this.onNewData);
+    },
+
+    onNewData: function(){
+        this.setState({
+            instanceHistory: stores.InstanceHistoryStore.fetchWhere({"instance": this.props.instance.id})
+        });
+    },
+
+    render: function () {
+        var instance = this.props.instance;
+        var content, items, deletedInfo;
+
+        if(instance.get('end_date')){
+            deletedInfo = <lh><strong>{"Deleted on " + moment(instance.get('end_date')).format("MMMM Do YYYY, h:mm a")}</strong></lh>
```

Comment:
Let's be consistent with the semi-colons here & on line 42 below.

Also, I would put parens around this and make it multi-line: 
```
deletedInfo = (
     <lh>
         <strong>
        {"Deleted on " + moment(instance.get('end_date')).format("MMMM Do YYYY, h:mm a")}
        </strong>
     </lh>
);
```